common/source/docs/common-speedybeef4-v3.rst

```diff
@@ -0,0 +1,120 @@
+.. _common-speedybeef4-v3:
+
+================
+Speedy Bee F4 V3
+================
+
+.. image:: ../../../images/speedybeef4-v3.png
+    :target: ../_images/speedybeef4-v3.png
+
+*above image and some content courtesy of the* `speedybee.com <https://www.speedybee.com>`__
+
+
+Specifications
+==============
+
+-  **Processor and Sensors**
+
+   -  STM32F405 ARM microcontroller
+   -  BMI20 IMU (Gyrp and Accelerometers)
+   -  DPS280 Barometer
+   -  AT7456E OSD
+
+-  **Interfaces**
+
+   -  9x PWM outputs (PWM9 for Neopixel LED)
+   -  1x RC input (PWM/PPM, SBUS)
+   -  6x serial port inputs (including RC input listed above)
+   -  1x I2C for external compass or airspeed sensor
+   -  Micro SD card slot
+   -  4 in 1 ESC connector
+   -  DJI Air Unit connector
+   -  USB-C connector
+
+-  **Power**
+
+   -  9V ~ 25V DC input power (3S-6S)
+   -  5V 2A BEC for peripheral 
+   -  9V 2A for Video
+
+-  **Size and Dimensions**
+
+   - 41.6mm x3 9.4mm x 7.8mm (30.5mm x 30.5mm mount pattern)
+   - 9.6g
+
+Pinouts
+=======
+
+.. image:: ../../../images/speedybeef4-v3-pinout-top.jpg
+    :target: ../../_images/speedybeef4-v3-pinout-top.jpg
+
+.. image:: ../../../images/speedybeef4-v3-pinout-bottom.jpg
+    :target: ../../_images/speedybeef4-v3-pinout-bottom.jpg
+
+UART Defaults
+=============
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the receive pin for UARTn. The Tn pin is the transmit pin for UARTn. N
+
+-    SERIAL0 -> USB
+-    SERIAL1 -> UART1 (DJI-VTX, DMA-enabled)
+-    SERIAL2 -> UART2 (RCIN, DMA-enabled) Note that the SBUS input must be used for SBUS.
+-    SERIAL3 -> UART3 (User defined)
+-    SERIAL4 -> UART4 (connected to internal BT module, not currently usable by ArduPilot)
+-    SERIAL5 -> UART5 (ESC Telemetry)
+-    SERIAL6 -> UART6 (GPS, DMA-enabled)
+
+RC Input
+========
+
+RC input is configured on the R2 (UART2_RX) pin for most RC unidirectional protocols except SBUS which should be applied at the SBUS pin. PPM is not supported. For Fport, a bi-directional inverter will be required. See `this article for connection information for F4 autopilots <https://ardupilot.org/plane/docs/common-connecting-sport-fport.html>`__. 
+
+For CRSF/ELRS/SRXL2 connection of the receiver to T2 will also be required.
+
+Dshot capability
+================
+
+The SpeedyBee F405 v3 supports up to 9 PWM outputs. The M1 to M4 outputs are on the esc connector, M5 to M8 on solder pads, plus LED pad(PWM output 9) for LED strip or another PWM output.
+
+The PWM is in 3 groups:
+
+-  PWM 1-4 in group1
+-  PWM 5-8 in group2
+-  PWM 9   in group3
+
+Channels within the same group need to use the same output protocol. If any channel in a group uses DShot then all channels in the group need to use DShot. Channels 1-4 also support bi-directional DShot.
```

Comment:
Currently you need to add that bi-dir dshot cannot be used in conjunction with I2C